packages/cardhost/app/routes/cards.js

```diff
@@ -6,6 +6,7 @@ export default class CardsRoute extends Route {
   queryParams = { editingCss: {} }
 
   model({ editingCss }) {
-    this.cssModeToggle.setEditingCss(editingCss);
+    let editingBool = editingCss === "true" ? true : false;
```

Comment:
The query param is cast to a string! So we need to turn it back into a boolean. Otherwise `"false"` is truthy and our if helpers in the template do the wrong thing.